src/PKSim.Presentation/Presenters/Simulations/ConfigureSimulationPresenter.cs

```diff
@@ -152,7 +160,12 @@ protected override void UpdateSimulationProperties()
 
          _simulationModelCreator.CreateModelFor(Simulation);
 
-         //now update all parameters from the orginal simulationToClone
+         //After the simulation was created, we can update the output mapping (we need the model to be available in the simulation)
+         //to make sure we swap out the simulation references
+         _originalSimulation.OutputMappings.Each(x => Simulation.OutputMappings.Add(x.Clone()));
```

Comment:
This is the code change here. When we change individual, or compound, we recreate a whole simulation completely. This clones the output and we have to swap the reference as well

Reply:
it's also working as expected when the compound is swapped. The path is still pointing to the old one, but it's marked as invalid and is shown as None in the UI
@georgeDaskalakis Do you see any problem with this?